api/src/experimental/index.ts

```diff
@@ -13,5 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+/**
+ * Any exports here may change at any time and without warning
+ * @module @opentelemetry/api/experimental
```

Comment:
these are needed to ensure that the module is named correctly in the generated documentation - the text above is added right underneath the heading.